csm_web/scheduler/tests/models/test_label.py

```diff
@@ -0,0 +1,150 @@
+import pytest
+from freezegun import freeze_time
+from unittest import mock
+import factory
+import faker
+import datetime
+from rest_framework import status
+import scheduler
+
+
+from django.utils import timezone
+from django.urls import reverse
+from scheduler.models import (
+    User,
+    Course,
+    Section,
+    Mentor,
+    Student,
+    Attendance,
+    SectionOccurrence,
+    Label
+)
+from scheduler.factories import (
+    UserFactory,
+    CourseFactory,
+    SectionFactory,
+    MentorFactory,
+    StudentFactory,
+    AttendanceFactory,
+    SpacetimeFactory,
+    LabelFactory,
+    CoordinatorFactory
+)
+
+import zoneinfo
+
+
+@pytest.fixture
+def setup_section(db):
+    mentor_user = UserFactory.create(
+        username="mentor_user", first_name="mentor", last_name="user"
+    )
+    student_user = UserFactory.create(
+        username="student_user", first_name="student", last_name="user"
+    )
+    coordinator_user = UserFactory.create(
+        username="coordinator_user", first_name="coordinator", last_name="user"
+    )
+    course = CourseFactory.create(
+        name="course",
+        title="title for course"
+    )
+    mentor = MentorFactory.create(user=mentor_user, course=course)
+    coordinator = CoordinatorFactory.create(user=coordinator_user, course=course)
+    labels = LabelFactory.create(
+
+        course=course
+    )
```

Comment:
Reference `batch_create` to create multiple labels at once: https://factoryboy.readthedocs.io/en/stable/reference.html#factory.Factory.create_batch

The tests should also cover cases where there are no labels.